aws/resource_aws_iam_role.go

```diff
@@ -285,6 +285,25 @@ func resourceAwsIamRoleDelete(d *schema.ResourceData, meta interface{}) error {
 				}
 			}
 		}
+		// For inline policies
+		inlinePoliciesResp, err := iamconn.ListRolePolicies(&iam.ListRolePoliciesInput{
```

Comment:
Aren't we going to miss some policies here potentially due to pagination? How do you feel about using `ListRolePoliciesPages` instead?